content/docs/command-reference/exp/apply.md

```diff
@@ -14,20 +14,21 @@ positional arguments:
 
 ## Description
 
-Restores the results of a given `experiment` (name or hash are accepted, see
-`dvc exp run` for details) into the workspace. This means changing the
-appropriate [metafiles](/doc/user-guide/project-structure),
-<abbr>parameter</abbr> files, <abbr>metrics</abbr>, <abbr>plots</abbr>, and
-corresponding DVC-tracked data.
+Restores a given `experiment` (name or hash are accepted, see `dvc exp run` for
+details) into the workspace. This means changing all files (code, data,
+<abbr>parameters</abbr>, <abbr>metrics</abbr>, etc.) needed to reflect the
+experiment conditions and results in the workspace.
 
-> This is similar to `dvc checkout`, but for `dvc experiments`.
+This command is typically used after finding a target `experiment` with
+`dvc exp show` or `dvc exp diff`, and before committing it to Git (making it
+[persistent](/doc/user-guide/experiment-management#persistent-experiments)).
 
-This is typically used after using `dvc exp show` or `dvc exp diff` to find the
-best experiment, and before committing to Git in order to make it
-[persistent](/doc/user-guide/experiment-management#persistent-experiments).
+Note that any [checkpoints](/doc/command-reference/exp/run#checkpoints) found in
+the `experiment` are **not** preserved when applying and committing it.
```

Comment:
Alternate suggestion (same idea as previous comment adding `intermediate`):
```suggestion
the `experiment` are **not** preserved in separate commits when applying.
```
Again, not a blocker, decide for yourself.